src/react/src/util/util.js

```diff
@@ -908,10 +908,15 @@ export const claimAFacilityFormIsValid = ({
         isEmpty(numberOfWorkers),
         NUMERIC_DASH_REGEX.test(numberOfWorkers),
     ]) &&
-    some([
-        !isEmpty(businessWebsite) && isURL(businessWebsite),
-        !isEmpty(businessLinkedinProfile) && isURL(businessLinkedinProfile),
-        !isEmpty(businessUploadFiles),
+    every([
+        isEmpty(businessWebsite) ||
```

Comment:
It's hard to understand the logic behind these conditions with multiple `||` and `&&` operators. Could you refactor this by replacing statements in the parenthesis into separate functions?

Reply:
I suppose to refactor it later on, as it didn’t bring any risk at the moment, only a visibility to read. @VadimKovalenkoSNF